content/projects/tdd/string-calculator/_index.md

```diff
@@ -205,4 +205,14 @@ add("1,2,3//;\n1000,1;2")
 
 ### why is this important?
 
-- If you're wondering to yourself, "Why is this sooo important!?" take a look at {{% contentlink path="/topics/data_validation_and_verification/" %}}
\ No newline at end of file
+- If you're wondering to yourself, "Why is this sooo important!?" take a look at {{% contentlink path="/topics/data_validation_and_verification/" %}}
+
+## Instructions For Reviewers
+
+- Look out for file structure. It might seem obvious, but people sometimes mess up the file structure of a TDD project. Refer to : **Set up your environment** which is at the top of this page.
+
+- Code should be clean, DRY and should follow the KISS(Keep It Simple, Stupid) principal - look out for these when reviewing. Also ensure that they have used a tool like Prettier or Black to format their code.
```

Comment:
The review instructions should be project specific.

Clean code conventions should not be mentioned in the review instructions as this relates to every project.